packages/terra-form-input/tests/jest/FormInput.test.jsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import FormInput from '../../src/Input';
```

Comment:
This file will need changes with updates in `packages/terra-form-input/src/Input.jsx` for exporting the component as `Input` instead of `FormInput`